heareval/tasks/coughvid.py

```diff
@@ -366,8 +366,24 @@ def run(self):
         labelcsv = csv.writer(
             open(os.path.join(self.workdir, "labelvocabulary.csv"), "wt")
```

Comment:
This could be a separate PR. It maps the labels in train and test to the vocabulary numbers for the final task output